.travis.yml

```diff
@@ -0,0 +1,41 @@
+language: android
+dist: trusty
+
+before_cache:
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+
+cache:
+  directories:
+    - $HOME/.gradle/caches/
+    - $HOME/.gradle/wrapper/
+
+jdk:
+  - oraclejdk8
+
+android:
+  components:
+    - build-tools-26.0.2
```

Comment:
This seems odd. Why did you choose 26 for buold tools and android version?

Reply:
Travis is stuck in time, I am trying to make it work. Tests require android SDK 28.